scripts/gitlab/check_line_width.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+#
+# check if line width of rust source files is not beyond x characters
+#
+
+
+BASE_BRANCH="origin/master"
+LINE_WIDTH="101"
+
+
+if git diff ${BASE_BRANCH}...${CI_COMMIT_SHA} \
+  | grep "^.\{${LINE_WIDTH}\}" ${file}
+then
+  echo "|"
+  echo "| newly added lines longer than ${LINE_WIDTH} characters."
```

Comment:
It is actually longer than 100 characters : P but I doooon't really care

Reply:
```suggestion
  echo "| newly added lines longer than $((LINE_WIDTH - 1)) characters."
```